frontend/apps/reader/modules/readerstatus.lua

```diff
@@ -1,38 +1,43 @@
 local BookStatusWidget = require("ui/widget/bookstatuswidget")
-local ButtonDialogTitle = require("ui/widget/buttondialogtitle")
+local ButtonDialog = require("ui/widget/buttondialog")
 local Device = require("device")
-local Event = require("ui/event")
-local FileManagerBookInfo = require("apps/filemanager/filemanagerbookinfo")
 local InfoMessage = require("ui/widget/infomessage")
 local UIManager = require("ui/uimanager")
 local WidgetContainer = require("ui/widget/container/widgetcontainer")
 local util = require("util")
 local _ = require("gettext")
 
 local ReaderStatus = WidgetContainer:extend{
-    document = nil,
-    enabled = true,
-    total_pages = 0,
+    title = _("Book status"),
 }
 
 function ReaderStatus:init()
-    if self.ui.document.is_pic then
-        self.enabled = false
-    else
```

Comment:
Does it actually work and/or do something for PicDocument?

Reply:
When a picture file is opened as a PicDocument, we get the same Book status window as when it is opened with MuPDF.